aws-ts-oidc-provider-pulumi-cloud/index.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import * as pulumiservice from "@pulumi/pulumiservice";
+
+// Configurations
+const audience = pulumi.getOrganization();
+const config = new pulumi.Config();
+const oidcIdpUrl: string = config.require("oidcIdpUrl");
+const thumbprint: string = config.require("thumbprint");
+const escEnv: string = config.require("escEnv");
+
+// Create a new OIDC Provider
+const oidcProvider = new aws.iam.OpenIdConnectProvider("oidcProvider", {
+    clientIdLists: [audience],
+    url: oidcIdpUrl, // Replace with your IdP URL
```

Comment:
```suggestion
    url: oidcIdpUrl,
```

They aren't replacing it since it comes from a config value.